hyperparams.yaml

```diff
@@ -1,4 +1,5 @@
 resnet18:
+  HYPERPARAM_SEARCH_ID: a29yoyfu-resnet18-160522_22:52
```

Comment:
I believe these were used to fetch the correct dataset. I would remove these unless they are still useful.